### PR TITLE
add support for QUIC 37

### PR DESCRIPTION
**Pull request by marten-seemann**

### CODE REVIEW

File: integrationtests/chrome_test.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ func init() {
 var _ = Describe("Chrome tests", func() {
 	It("does not work with mismatching versions", func() {
 		versionForUs := protocol.SupportedVersions[0]
-		versionForChrome := protocol.SupportedVersions[len(protocol.SupportedVersions)-1]
+		versionForChrome := protocol.SupportedVersions[1]
 
 		// If both are equal, this test doesn't make any sense.
 		if versionForChrome == versionForUs {
```
Comment (lucas-clemente) [inline integrationtests/chrome_test.go]: This check can be removed with your above change.

Comment (marten-seemann) [inline integrationtests/chrome_test.go]: I was hoping that the above change is only temporary and we can remove it either by updating the Docker image or merging your Chrome-branch.

Comment (lucas-clemente) [inline integrationtests/chrome_test.go]: The change doesn't worsen the ability of this test to detect an improper testing setup, so I don't see why it should be temporary.

Comment (marten-seemann) [inline integrationtests/chrome_test.go]: Got it. I'll remove it.

**Pull request merged**